content/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-curl-and-postman.md

```diff
@@ -0,0 +1,333 @@
+
+![blog figure9](/img/greenlake-com-ops-api-curl16.png)
+
+To learn more about Filtering, see [the HPE GreenLake for Compute Ops Management guide.](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/guide/#filtering)
+
+The HPE GreenLake for Compute Ops Management API supports many resources that are described in the [API reference](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/openapi/compute-ops-latest/overview/) document, as illustrated below:
+
+![blog figure10](/img/greenlake-com-ops-api-curl17.png)
+
+Unique Resource Identifiers [(URIs)](<>)  are used to identify a resource. A URI is a full API path ending in an identification number. 
+
+For example:
+
+* compute-ops/v1beta2/servers/{serverId}
+* compute-ops/v1beta1/reports/{id}/data
+
+**v1beta1**, **v1beta2** in the URI is the version of the resource that is being accessed.
+
+One can invoke the common HTTP [](<>)methods, like GET, POST, PUT, PATCH, and DELETE, on resources in the HPE GreenLake for Compute Ops Management API as shown below for the **filters** resource:
+
+![blog figure11](/img/greenlake-com-ops-api-curl23.png)
+
+Refer to the [API reference](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/openapi/compute-ops-latest/overview/) site for a complete list of [](<>)methods supported by each API resource.
+
+# Using Postman
+
+Postman is a tool designed to build and use APIs.
+
+To get started, “create a request”, as shown below:
+
+![blog figure12](/img/greenlake-com-ops-api-curl7.png)
+
+In the request URL field, enter the endpoint URL: ﻿
+
+[](<>)**<https://us-west2-api.compute.cloud.hpe.com/compute-ops/v1beta2/servers?limit=2>**
+
+This is the base connectivity endpoint you have seen in earlier, followed by the resource URI you want to query ﻿**/compute-ops/v1beta2/servers** as described in the API reference table.
+
+![blog figure13](/img/greenlake-com-ops-api-curl25.png)
+
+In order to limit the output as documented in the request pane of the *servers* resource:
+
+We use query parameter ﻿**?limit=2**
+
+![blog figure14](/img/greenlake-com-ops-api-curl19.png)
+
+![blog figure15](/img/greenlake-com-ops-api-curl20.png)
+
+In the *Authorization* tab, choose  **Bearer Token** in the *Type* drop-down menu and paste the access token that was generated earlier in the Token field. Postman will generate the appropriate Authorization header when you send the request.
+
+![blog figure16](/img/greenlake-com-ops-api-curl8.png)
+
+Hit the **Send** button to get a **200 OK** status response indicating success and a JSON body with the details of two compute servers on boarded and assigned to the corresponding application for the user's account.
```

Comment:
on boarded should be "on-boarded"

![image](https://user-images.githubusercontent.com/77496261/205347403-9097a91c-5102-4fb6-97c1-0c51f477d658.png)

Reply:
done